app/models/host/base.rb

```diff
@@ -120,6 +120,7 @@ def import_facts(facts)
 
       facts[:domain].try(:downcase!)
 
+      hostgroup_title = facts.delete(:foreman_hostgroup)
```

Comment:
This fact isn't standard, from what I read on the issue description it's a custom fact. I think the functionality is neat, but please add docs to this after it's merged